docs/dev/openid.md

```diff
@@ -20,6 +20,10 @@ Then, you need to configure these environment variables:
 - `OPID_CLIENT_SECRET` (*play container*): the secret of the OpenID client that you created in the OpenID Connect provider
 - `OPID_PROMPT` (*play container*): whether the Authorization Server prompts the End-User for reauthentication and consent. Used as the [`prompt` parameter of the authentication request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) (Default: login)
 
+There are additional environment variables which can be used to configure the OpenID login process further:
+- `OPID_USERNAME_CLAIM` (*play container*): the claim attribute to be used as the username on login. (Default: username)
```

Comment:
Currently documented this one as is. Maybe another a discussion for a separate PR could be changing the default value of the `OPID_USERNAME_CLAIM` variable. Currently it defaults to `username` which is not one of the standard claim names: https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
Might make sense to use one of the standard claims so that the system works out of the box. (We needed to investigate here a little bit after a successful login as to why we were still getting asked to set a username; our OpenID provider did not send a username field).

Reply:
> Might make sense to use one of the standard claims so that the system works out of the box

Gee, I never noticed that before. You are correct.
Would you mind opening a different PR for that? This way, I can merge this one right now and take a bit more time to prepare for the impact of the other PR?

Reply:
@moufmouf thanks a lot for the very fast feedback and the merges. Ok, sure, I will open another PR soon for selecting a default value for the OIDC name, we can continue the discussion there.